cirq-superstaq/cirq_superstaq/job_test.py

```diff
@@ -92,11 +92,11 @@ def test_num_qubits(job: css.job.Job) -> None:
 
     # The first call will trigger a refresh:
     with mocked_get_job_requests(job_dict) as mocked_get_job:
-        assert job.num_qubits() == {"job_id": {"num_qubits": 2}}
+        assert job.num_qubits() == [2]
```

Comment:
Why is this returning `List[int]` instead of `int`?

Reply:
Because a user could submit multiple circuits with different `num_qubits` in a single job. In that case, calling `num_qubits()` for the job should list the number of qubits for each circuits individually as a list

Reply:
> Does qiskit store multiple circuits in a Job?

yep - this is why `qss.SuperstaqJob` does as well (which this is attempting to achieve parity with)

Reply:
> > Does qiskit store multiple circuits in a Job?
> 
> yep - this is why `qss.SuperstaqJob` does as well (which this is attempting to achieve parity with)

^^